src/parser/shared/modules/Haste.js

```diff
@@ -35,6 +35,7 @@ class Haste extends Analyzer {
     [SPELLS.QUICK_THINKER_BUFF.id]: 0.15,
     [SPELLS.EMPOWER_RUNE_WEAPON.id]: 0.15, // Frost DK
     [SPELLS.EVER_RISING_TIDE_CHARGING_BUFF.id]: 0.1, // Essence
+    [SPELLS.GUARDIAN_OF_AZEROTH_HASTE_BUFF.id]: 0.02, // Essence
```

Comment:
This is per stack right? So it should be 
```
[SPELLS.GUARDIAN_OF_AZEROTH_HASTE_BUFF]: {
  hastePerStack: 0.02
}
```

Reply:
yep, thanks. I didnt know how to add a stacking haste buff so i added it like that temporarily.